bionic/core/task_execution.py

```diff
@@ -751,13 +751,34 @@ def _local_artifact_from_value(self, value):
                 )
             ) from e
 
-        value_hash = protocol.tokenize_file(value_path)
+        value_hash = protocol.tokenize_file(
```

Comment:
nit: Aesthetically, it feels to me like hash strings should have a fixed, moderate length, so I'd be tempted to create a new hash from these two values instead of concatenating them. (Although there's no functional benefit except for making the metadata YAML a tiny bit cleaner-looking.) WDYT?

Reply:
Yeah, that works for me. I'll hash them using `hash_simple_obj_to_hex`.